Hymns/Infrastructure/Data/PdfPreloader.swift

```diff
@@ -33,6 +33,7 @@ class PDFLoader {
     }
 
     func get(url: URL) -> PDFDocument? {
+        analytics.logDisplayMusicPDF(url: url)
```

Comment:
If you are logging displaying the pdf, I think it makes more sense for it to be in the PDF Viewer itself, rather than in the cache. In the future, getting something from the cache,  doesn't necessarily mean you're going to immediately be displaying the pdf